File: src/main/java/org/artembogomolova/demo/webapp/dao/repo/core/IPersonRepository.java
package org.artembogomolova.demo.webapp.dao.repo.core;

import org.artembogomolova.demo.webapp.domain.core.Person;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface IPersonRepository extends CrudRepository<Person, Long> {

}

File: app/src/main/kotlin/org/artembogomolova/demo/webapp/main/dao/repo/IPersonRepository.kt
package org.artembogomolova.demo.webapp.main.dao.repo

import org.artembogomolova.demo.webapp.main.domain.core.Person
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface IPersonRepository : CrudRepository<Person?, Long?>